packages/blockchain-wallet-v4-frontend/src/modals/Btc/SendBtc/FirstStep/template.success.js

```diff
@@ -209,12 +209,19 @@ const FirstStep = props => {
             />
             {enableToggle ? (
               !toToggled ? (
-                <AddressButton onClick={() => handleToToggle()}>
+                <AddressButton
+                  onClick={() => handleToToggle()}
+                  data-e2e='downArrowButton'
+                >
                   <Icon name='down-arrow' size='11px' cursor />
                 </AddressButton>
               ) : (
-                <AddressButton onClick={() => handleToToggle()}>
-                  <Icon name='pencil' size='13px' cursor />
+                <AddressButton
+                  onClick={() => handleToToggle()}
+                  cursor
```

Comment:
Do you want to add cursor to the AddressButton instead of the Icon pencil for some reason?

Reply:
I only added 2 data-e2e tags and didn't do any other changes

Reply:
The cursor was removed from <Icon name='pencil' size='13px' cursor /> and added to  <AddressButton.

Can you just put the `cursor` back to `<Icon name='pencil' size='13px' />` and remove from AddressButton (line 221)

Reply:
Oh, my bad.

Reply:
Thanks!